includes/class-coblocks-form.php

```diff
@@ -897,6 +904,66 @@ public function process_form_submission( $atts ) {
 
 	}
 
+	/**
+	 * Setup the email subject line, replacing any found shortcodes.
+	 * Note: [email] will be replaced with the value of field-email
+	 *       [name] will be replaced with the value of field-name etc.
+	 *
+	 * @param  array  $atts           Block attributes array.
+	 * @param  string $email_field_id Email field ID.
+	 * @param  string $name_field_id  Nane field ID.
+	 * @return string Email subject.
+	 */
+	private function setup_email_subject( $atts, $email_field_id, $name_field_id ) {
+
+		$default_subject = __( 'Form submission from [email]', 'coblocks' );
```

Comment:
@AnthonyLedesma It would be ideal if we could set this in a class property or something so that we're not re-defining it. It's also defined here https://github.com/godaddy-wordpress/coblocks/pull/1017/files#diff-6889c8f6a940ab86e23a2cf5809324ccR114

Reply:
Also, the `[email]` part of the string should not be translatable. I'd use `sprintf` so we can add placeholders here. There are examples in the codebase already.

Reply:
These have both been changed.